MANIFEST.in

```diff
@@ -8,9 +8,54 @@ include yt/utilities/mesh_types.yaml
 exclude yt/utilities/lib/cykdtree/c_kdtree.cpp
```

Comment:
This bit is included below (see line 40).
```suggestion
```

Reply:
oh yeah, good catch. I have no idea why but this file is actually empty so it probably doesn't hurt to include it back.
@Xarthisius wrote the exclude line in #3403, so let's give him a chance to comment.

Reply:
from that PR's description:

> excludes that I added are mostly cosmetic

Reply:
OK, whatever you decide I'm happy with. It's not as if adding an empty cpp file would significantly increase the archive size.